src/System.Data.Odbc/src/System/Data/Odbc/odbcmetadatacolumnnames.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It'd be nice if the file name was cased correctly, e.g. OdbcMetaDataColumnNames.cs